src/_js/map.js

```diff
@@ -181,6 +181,43 @@
 			}
 		});
 	};
+	TibiaMap.prototype._loadMarkers = function () {
+		const _this = this;
+		const icons = []
+		const symbols = ["!", "$", "?", "bag", "checkmark", "cross", "crossmark", "down", "flag", "lock", "mouth", "red down", "red left", "red right", "red up", "skull", "spear", "star", "sword", "up"];
+		symbols.forEach(s => {
+			icons[s] = L.icon({iconSize: [11, 11], iconUrl: s.replace("!", "exclamation").replace("$", "dollar").replace("?", "question").replace(" ", "_") + ".png"});
+		});
+
+		const xhr = new XMLHttpRequest();
+		xhr.open('GET', 'https://tibiamaps.github.io/tibia-map-data/markers.json', true);
+		xhr.responseType = 'json';
+		xhr.onload = function () {
+			if (xhr.status === 200) {
+				xhr.response.forEach(m => {
+					let options = {'title': m.description};
+					if (m.icon && m.icon in icons) { options.icon = icons[m.icon]; }
+					if (!_this.markersLayers[m.z]) { _this.markersLayers[m.z] = new L.layerGroup(); }
+					_this.markersLayers[m.z].addLayer(L.marker(_this.map.unproject([m.x + 0.5, m.y + 0.5], 0), options));
+				});
+				_this._tryShowMarkers();
+			}
+		};
+		xhr.send();
+	};
+	TibiaMap.prototype._toggleMarkers = function () {
+		this.showMarkers = !this.showMarkers;
+		this._tryShowMarkers()
+	}
+	TibiaMap.prototype._tryShowMarkers = function () {
+		const _this = this;
+		this.markersLayers.forEach((l,f) => {
```

Comment:
The code gets minified at build time, so we don't need to do it ourselves :) Let's use descriptive names, i.e. `layer` instead of `l` and `floor` instead of `f`.